### PR TITLE
Add kulkarnigaurav38 profile

### PR DESCRIPTION
**Pull request by kulkarnigaurav38**
# Please Merge my Profile file !
## Just getting started here with open-source contributions!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/334"></a>

### CODE REVIEW

**Review by niteshseram — APPROVED** [review]
LGTM

**Review by CodingSpecies — APPROVED** [review]

**Review by aayush89890 — COMMENTED** [review]
Looks good, Welcome to the community @kulkarnigaurav38 .

File: public/data/kulkarnigaurav38.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Gaurav Kulkarni",
+  "bio": "Undergrad || CSE || Learning Full-Stack Development",
+  "avatar": "https://github.com/kulkarnigaurav38.png",
+  "links": [
+    {
+      "name": "Checkout my GitHub profile",
```
Comment (Panquesito7) [inline public/data/kulkarnigaurav38.json:7]:
```suggestion
      "name": "Check out my GitHub profile",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — APPROVED** [review]
Thanks

**Pull request merged**